projects/elife.yaml

```diff
@@ -121,6 +121,7 @@ defaults:
             ec2:
                 ami: ami-9eaa1cf6 # Ubuntu 14.04
     vagrant:
```

Comment:
would include a reason why this is better than the Canonical one (long-standing bug you posted a while ago)